src/main/kotlin/org/taktik/icure/be/ehealth/logic/kmehr/v20161201/KmehrExport.kt

```diff
@@ -382,6 +399,18 @@ open class KmehrExport {
         }
     }
 
+    fun makeText(language: String, content: Content): TextType?{
+        return (content.booleanValue ?: content.numberValue ?: content.stringValue ?: content.instantValue
+        ?: content.measureValue ?: content.medicationValue ?: content.binaryValue ?: content.documentId).let {
+            TextType().apply {
+                l = language
+                content.medicationValue?.compoundPrescription?.let {
+                    value = content.medicationValue?.medicinalProduct?.intendedname
+                }
+            }
+        }
+    }
+
```

Comment:
This is weird... You check all the contents to finally only select medicationValue... Isn't the following more what you want

```suggestion
    fun makeText(language: String, content: Content): TextType?{
        return (content.medicationValue?.compoundPrescription ?: content.medicationValue?.medicinalProduct?.intendedname)?.let {
            TextType().apply {
                l = language
                value = it
            }
        }
    }

```

Reply:
ok that's better